core/src/test/resources/Migrator/Ibisconfig_4_unittests_changeset.xml

```diff
@@ -22,4 +22,19 @@
 		</createTable>
 		<addPrimaryKey tableName="IBISCONFIG" columnNames="NAME, VERSION" constraintName="PK_IBISCONFIG"/>
 	</changeSet>
-</databaseChangeLog>
\ No newline at end of file
+
+	<changeSet id="IAF_Util:6" author="Yannick van Leeuwen">
+		<validCheckSum>7:85bdc937b35ee7e69753df403f970e82</validCheckSum> <!-- #1481, BLOB->LONGBLOB -->
+		<comment>Allow 120 chars for version field</comment>
+		<modifyDataType
+				columnName="VERSION"
+				newDataType="java.sql.Types.VARCHAR(120)"
+				tableName="IBISCONFIG"/>
+		<rollback>
+			<modifyDataType
+					columnName="VERSION"
+					newDataType="java.sql.Types.VARCHAR(50)"
+					tableName="IBISCONFIG"/>
+		</rollback>
+	</changeSet>
```

Comment:
Voor de test maakt het niet zo veel uit hoor, die databases worden (vaak/ als het goed is) steeds opnieuw gemaakt 😄